config/manifests/bases/glance-operator.clusterserviceversion.yaml

```diff
@@ -6,6 +6,11 @@ metadata:
     capabilities: Basic Install
     features.operators.openshift.io/disconnected: "true"
     features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "false"
```

Comment:
I'm not sure about `tls-profiles` yet.

Reply:
looks like the recommendation is to keep them to false until we have the opportunity to check and eventually change the bool values. In this case I think we're good to go.